lte/gateway/c/core/oai/tasks/amf/amf_Security_Mode.cpp

```diff
@@ -90,10 +90,11 @@ int amf_handle_security_complete_response(
       }
     }
 
-    /* FSM takes care of sending initial context setup request */
-    ue_state_handle_message_initial(COMMON_PROCEDURE_INITIATED1,
-                                    STATE_EVENT_SEC_MODE_COMPLETE, SESSION_NULL,
-                                    ue_mm_context, amf_ctx);
+    // s6a update location request
+    int rc = amf_send_n11_update_location_req(ue_mm_context->amf_ue_ngap_id);
```

Comment:
Why rc is re-declared? As it is already initialized at the beginning of this fun().

Reply:
addressed